types/json/binary_functions.go

```diff
@@ -647,6 +647,16 @@ func compareInt64(x int64, y int64) int {
 	return 1
 }
 
+func compareFloat64(x float64, y float64) int {
```

Comment:
Can we use compareFloat64PrecisionLoss

Reply:
The origin code used it. The got the wrong result.